contributing.md

```diff
@@ -107,8 +107,23 @@ the rest of the rebase.
 
 ## Creating a Pull Request
 
+1. Set up SSH if you haven't already, or generate a new SSH key if necessary as described in the [Github Documentation](https://docs.github.com/en/authentication/connecting-to-github-with-ssh)
 1. Fork the Github repository at https://github.com/elyra-ai/xxx if you haven't already.
 1. Clone your fork, create a new branch, push commits to the branch.
+     * Below is a vague outline of the commands to be followed when doing these steps:
+          * Cloning your fork:
+               * `git clone git@github.com:elyra-ai/<desired-elyra-repository>.git`
+               * Set `upstream` as described in the [GitHub documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/configuring-a-remote-for-a-fork), but using SSH instead of HTTPS
+          * Creating a new branch:
+               * `git checkout -b <branch-name>`
+          * Committing and pushing a file:
+               * `git add <file-name>`
+               * `git commit`
+                    * Opens a text file that is a summary of the current commit operation
+                    * Add a commit message that describes the changes you made
+                    * Ctrl + X and enter to save
+               * `git push`
+                    * Pushes the commit from your local repo to the remote repo
```

Comment:
I think this section could probably be made a bit less specific since there are several different options for the `git` `add`, `commit`, and `push` commands. 

For each of these three commands, why don't we remove all the sub-bullets and just have a single sentence about the 'purpose' of the command that also includes a link to its man page. For example, maybe instead of the bullet point that says "`git add <file-name>`" you could say 

```
* Make changes to the necessary files, then save them with [`git add`](https://git-scm.com/docs/git-add)
```

One final note, and this is nothing that you introduced, is that I think we should move that entire step 8 block below ("Follow The 7 rules for a great commit message") to right under the `commit` bullet point. It probably makes more sense for new users to check out that article as they're following along with these steps.